source/api/presentation/3.0/index.md

```diff
@@ -531,27 +531,48 @@ The value _MUST_ be a string.
 
 A set of user experience features that the publisher of the content would prefer the client to use when presenting the resource. This specification defines the values in the table below. Others may be defined externally as an [extension][prezi30-ldce].
 
+In order to determine the behaviors that are governing a particular resource, there are four inheritance rules:
+* Collections inherit behaviors from their parent Collection.
+* Manifests **DO NOT** inherit behaviors from any parent Collections.
+* Canvases inherit behaviors from their parent Manifest, but **DO NOT** inherit behaviors from any parent Ranges.
+* Ranges inherit behaviors from their parent Range and parent Manifest.
+
+When interpreting the Layout Behaviors and `auto-advance` for Ranges, they govern the rendering of the resources when that particular Range is selected but are not always applied.  A Range with the `behavior` value `continuous`, in a Manifest with the `behavior` value `paged`, would mean that the Manifest's Canvases should be rendered in a paged fashion, unless the range is selected to be viewed, and its included Canvases would be rendered in that context only as being virtually stitched together. This might occur, for example, when a physical scroll is cut into pages and bound into a codex with other pages, and the publisher would like to provide the user the experience of the scroll in its original form.
+
+The descriptions of the behavior values have a set of which other values they are disjoint with, meaning that the same resource _MUST NOT_ have both of two or more from that set. In order to determine which is in effect, the client _SHOULD_ follow the inheritance rules above, taking the value from the closest resource. The user interface effects of the possible permutations of non-disjoint behavior values are client dependent, and implementers are advised to look for [relevant recipes][annex-cookbook] in the IIIF cookbook. 
```

Comment:
Seems odd to hyperlink "relevant recipes" instead of "IIIF cookbook" -- I would expect the former to take me to the relevant ones.

(otherwise looks good)

Reply:
True! Fixed :)